### DISCUSSION TITLE
アップロードに失敗しました ファイルサイズの制限を超えてファイルをアップロードできません。

### DISCUSSION THREAD

**Discussion by zxcv-12345** [misskey-dev/misskey#13983] [General]
misskeyの中でどのようにファイルアップロードのサイズの制限を修正しますか?

私はプロファイルの中の
# Upload or download file size limits (bytes)
maxFileSize: 262144000

変更しました
maxFileSize: 4121440000

この問題に変わりはありません

**Comment by yszkst** [comment]
初期設定ではユーザーのネットドライブの容量は100MBです。
ユーザーに割り当てられているロールにより変更可能です。    

控制面板 => 角色 => 基本各色 => 网盘容量


これで解決しない場合は問題を特定するためにいくつか確認しなければならないことがあります


## 操作
+ アップロードしようとしたファイルのサイズ
+ エラーメッセージ
  + サーバーアプリケーションログから確認できるメッセージ
  + ブラウザの開発者ツールで確認できるメッセージ

## ファイル保存場所

+ ローカルのディスク(初期設定)
+ オブジェクトストレージ
   + サービス名とプラン

## リバースプロキシ

-  使用していない
-  ローカルインストールのApache/Nginx
-  Cloudflare Tunnel
- その他


## 環境 (NODE_ENV)

- Dev  (via "pnpm dev")
- Production   (via "pnpm start")
